### ISSUE TITLE
How to create an issue.

### ISSUE THREAD

**Issue by mhiaxo** [valehq/changelog#5]
Let's say I want to create an issue for setting up Duels, which is an important plugin for the PVP aspect of our server, and it might require a discussion when it's setup. 

I would click on the "Issues" tab and then click on "New Issue." In the Title and comment field, I would describe the task I am doing or the issue. We're not expecting any crazy long explanations or anything. Just like Trello, you can make a sweet and quick comment of the issue and submit it. 

The goal is to just create an issue or reference of changes on the server, so we can create a changelog at the end of the month and keep our players updated!

So,  this is what an issue would look like

![image](https://user-images.githubusercontent.com/61203966/77285754-250aed00-6ca8-11ea-82da-e3e853d4d6bd.png)

-  I created a checkmark list that I can edit when a task is done, and I explained what the issue/task is.  
- I've also uploaded images to the issue.
- I assigned the issue to myself as it's my intention to do it. I also attached a label as "To do", that the task needs to be done soon. If I didn't think the issue was too important, I'd add the label "Backlog."
- Not too hard, is it? 

Once the task is submitted and I want to check an item on the list, I can just click on the box

![image](https://user-images.githubusercontent.com/61203966/77285990-be3a0380-6ca8-11ea-9b6f-7c3122427201.png)

Once the issue is done, I can change the label to "Completed" or "Fixed".

![image](https://user-images.githubusercontent.com/61203966/77286099-f5a8b000-6ca8-11ea-86a7-8674aef4a0ec.png)

**When should I create an issue?**

- If you're making any sort of change to plugins, even a configuration value change, adding a plugin, disabling one, etc there needs to be an issue that's references this change. This should be labeled as "Changelog."
- If you're doing a lot of changes and they're all unrelated, such as modifying different plugins or doing a big plugin update, you can create a single issue listing them. But if any of them are huge and deserve it's own conversation, create an issue for it.  Also label this as ""Changelog."
- If you're fixing issues that are referenced here, you can update the label as "Fixed." or "Completed." (If it's not a bug.)  and close the issue. 
- If you'd like to discuss a suggestion or feedback that a player has posted on the forums or discord, this can also be an issue.
- If you've found any issues/bugs or if there is any issue that needs to be fixed, discussed, etc, that's an issue.
